### ISSUE TITLE
failes with chef 12.11.18 from chefdk 0.15.16

### ISSUE THREAD

**Issue by drspeedo** [thbishop/knife-solo_data_bag#36]
Chef 12.11.18 from ChefDK 0.15.16 appear not to work with the most recent version of knife solo data bag.  It doesn't see any data bag that isn't on the server and running knife solo data bag show doesn't display the contents of each .json file (like it does on older version) but shows the name of each data bag on the server along with its path in the server.

**Comment by CammoKing** [comment]
I can confirm that I have experienced this as well with an upgrade to ChefDK:
Chef Development Kit Version: 0.16.28
chef-client version: 12.12.15
delivery version: master (921828facad8a8bbbd767368bfc72f19bd30e7bd)
berks version: 4.3.5
kitchen version: 1.10.2

**Comment by joeyates** [comment]
@drspeedo I've updated the README to indicate that with Chef 12.x, you need to add `local_mode true` to your `.chef/chef.rb`. Could you check if that solves your problem?

**Comment by drspeedo** [comment]
@joeyates sorry it took so long to get back to you.  It doesn't .. adding it to .chef/chef.rb doesn't change anything.  Adding it to .knife/chef.rb or to the command line with --local-mode returns the error

ERROR: The object you are looking for could not be found
Response: Object not found: chefzero://localhost:8889/data/postgresql/default

how I'm running the command from older versions is in my data bag folder I run 

knife solo data bag show databag-name databag-item default --local-mode --data-bag-path ./ -Fj --secret-file ~/.chef/encrypted_data_bag_secret

**Comment by grv87** [comment]
I have the same problem with `--data-bag-path`.
`create` works:
```
knife solo data bag create default test --data-bag-path=src/kitchen/data_bags --local-mode
<editor starts>
```

But `show` and `edit` don't:
```
knife solo data bag edit default test --data-bag-path=src/kitchen/data_bags --local-mode
ERROR: The object you are looking for could not be found
Response: Object not found: chefzero://localhost:8889/data/default/test
```

I have `local_mode true` in both `~/.chef/knife.rb` and `~/.chef/chef.rb`.

**Comment by joeyates** [comment]
I've released version 2.1.0 after merging #40.
Please check whether that fixes this issue.

**Comment by joeyates** [comment]
Closing as the issue should be fixed in 2.1.0
